crypto/cms/cms_smime.c

```diff
@@ -882,14 +882,17 @@ int CMS_final(CMS_ContentInfo *cms, BIO *data, BIO *dcont, unsigned int flags)
         return 0;
     }
 
-    ret = SMIME_crlf_copy(data, cmsbio, flags);
+    SMIME_crlf_copy(data, cmsbio, flags);
```

Comment:
The return value should be checked.

Reply:
`git grep SMIME_crlf_copy` shows no check for return value all over, do you want to fix this in a separate patch for all locations or just this one? the only fail path for this is the BIO_new

Reply:
Added a separate commit to handle the `SMIME_crlf_copy` result code cross project